sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -433,7 +435,15 @@ public String getEntityPath() {
     public Mono<ServiceBusReceivedMessage> receiveDeferredMessage(long sequenceNumber) {
         return connectionProcessor
             .flatMap(connection -> connection.getManagementNode(entityPath, entityType))
-            .flatMap(node -> node.receiveDeferredMessage(receiveMode, sequenceNumber));
+            .flatMap(node -> node.receiveDeferredMessage(receiveMode, sequenceNumber))
+            .map(receivedMessage -> {
+                if (receiveMode == ReceiveMode.PEEK_LOCK && !CoreUtils.isNullOrEmpty(receivedMessage.getLockToken())) {
```

Comment:
It won't be zero lock token. It'll be null or empty. They had zero lock token because they used uuid before.

Reply:
So you don't need to do the conversion to uuid from string